lib/adapters/tool/base.ts

```diff
@@ -0,0 +1,73 @@
+import Testplane, {type Config, type TestCollection} from 'testplane';
```

Comment:
I create a separate folder `adapters` so that all adapters are in one place. I'm also going to move `TestResultAdapter` in one of the following PR and create new adapters - `Test`, `Config` (maybe something else)

Reply:
> I suggest to not import "testplane" in base tool adapter (playwright will be inherited from this class, right?)

I deleted this base class because after implement playwright tool adapter I understand that I only need only an interface. But inside this interface types from tesplane still in use. I will get rid of them in the next PR in which I will add the remaining adapters (TestCollection, Test and Config)